analyzer/src/main/kotlin/Analyzer.kt

```diff
@@ -133,9 +134,15 @@ class Analyzer(private val config: AnalyzerConfiguration, private val labels: Ma
                 async {
                     val results = manager.resolveDependencies(files, labels)
 
-                    // By convention, project ids must be of the type of the respective package manager.
+                    // By convention, project ids must be of the type of the respective package manager. An exception
+                    // for this is Flutter with Pub, which internally calls Gradle.
                     results.projectResults.forEach { (_, result) ->
-                        val invalidProjects = result.filter { it.project.id.type != manager.managerName }
+                        val invalidProjects = result.filterNot {
+                            val typeById = it.project.id.type
```

Comment:
What does the "ById" suffix refer to? How about just "projectType"?

Reply:
I wanted to differentiation between the "type" from `manager.managerName` and the type from `it.project.id.type`.
But thinking about it again `managerName` <-> `projectType` is better.